build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -520,6 +520,49 @@ message Platform {
   // The server MAY use the `value` of one or more properties to determine how
   // it sets up the execution environment, such as by making specific system
   // files available to the worker.
+  //
+  // The following standard `name`s are defined:
+  //
+  // * `OS`: This describes the operating system. Only a single value is allowed
+  //   and an exact match is required. The following standard values are
+  //   defined:
```

Comment:
Do you want to proscribe what should happen with non-standard values? One option is that clients and servers are free to make up their own; another is that anyone wanting to support a value not in the canonical list should namespace it in some way (e.g. '<service>_<os>').

Depends if we think that adding new standard values will be benign or problematic if some implementers are already using their own custom ones. (I'm leaning towards thinking it'll be benign, and so allowing free use of non-standard values, but interested in your take).

Reply:
Is it worth saying that this is, when possible, the result of running `uname -s` or is that not a correct approximation? For example my Mac reports Darwin which is not on the list but Linux, AIX and SunOS report as listed.

Reply:
I'm also thinking that namespacing non-standard values is not required. It may be different for other platform properties but for `os` and also `isa` I consider the risk of conflicting value definitions relatively low. There is a chance that implementations use different non-standard values for the same OS or ISA, however, namespacing non-standard values wouldn't help in that case.

> Is it worth saying that this is, when possible, the result of running `uname -s` or is that not a correct approximation? For example my Mac reports Darwin which is not on the list but Linux, AIX and SunOS report as listed.

I think it makes sense to match `uname -s` in most cases, but there are some exceptions. Besides Windows as special case without `uname`, I intentionally used 'MacOS' instead of 'Darwin' as I think the kernel alone is insufficient in that case. To my knowledge, Darwin doesn't have a stable syscall ABI and the common case would be for input binaries to depend on macOS userspace parts as well. Linux is different in this regard.

Reply:
I've added a brief comment about non-standard values. Does this suffice or any further suggestions on this?